src/routes/submit/submit-payload-schema.js

```diff
@@ -0,0 +1,74 @@
+import Joi from 'joi'
+
+const FileAnswerSchema = Joi.object({
+  type: Joi.string().valid('file').required(),
+  value: Joi.object({
+    skipped: Joi.boolean().required(),
+    path: Joi.string().optional()
```

Comment:
I didn't capture it in the original value, but if the file **isn't** skipped, it would be ideal if this is mandatory. 

The real type should have been:

```typescript
type FileAnswer = { skipped: true } | { skipped: false, path: string }
```

Reply:
I'll add this, good spot

Reply:
Done